adaptors/satusehat.md

```diff
@@ -0,0 +1,184 @@
+
+The following example shows a HTTP POST request to creating an `Encounter` FHIR
+resource. Data was taken from the
+[Satusehate Postman Collection](https://www.postman.com/satusehat/satusehat-public/request/56uan96/encounter-create)
+
+```js
+post('Encounter', {
+  resourceType: 'Encounter',
+  status: 'arrived',
+  class: {
+    system: 'http://terminology.hl7.org/CodeSystem/v3-ActCode',
+    code: 'AMB',
+    display: 'ambulatory',
+  },
+  subject: {
+    reference: 'Patient/100000030009',
+    display: 'Budi Santoso',
+  },
+  participant: [
+    {
+      type: [
+        {
+          coding: [
+            {
+              system:
+                'http://terminology.hl7.org/CodeSystem/v3-ParticipationType',
+              code: 'ATND',
+              display: 'attender',
+            },
+          ],
+        },
+      ],
+      individual: {
+        reference: 'Practitioner/N10000001',
+        display: 'Dokter Bronsig',
+      },
+    },
+  ],
+  period: {
+    start: '2022-06-14T07:00:00+07:00',
+  },
+  location: [
+    {
+      location: {
+        reference: 'Location/b017aa54-f1df-4ec2-9d84-8823815d7228',
+        display:
+          'Ruang 1A, Poliklinik Bedah Rawat Jalan Terpadu, Lantai 2, Gedung G',
+      },
+    },
+  ],
+  statusHistory: [
+    {
+      status: 'arrived',
+      period: {
+        start: '2022-06-14T07:00:00+07:00',
+      },
+    },
+  ],
+  serviceProvider: {
+    reference: 'Organization/{{Org_id}}',
+  },
+  identifier: [
+    {
+      system: 'http://sys-ids.kemkes.go.id/encounter/{{Org_id}}',
+      value: 'P20240001',
+    },
+  ],
+});
+```
+
+Checkout Satusehat's
+[Postman Collection](https://www.postman.com/satusehat/satusehat-public/overview)
+for more examples and resources.
+
+## Mapping Satusehat URLs to Adaptor Functions
+
+The Satusehat adaptor functions provide convenient HTTP method helpers - like
+[`get()`](/adaptors/packages/satusehat-docs#get).
+
+Each takes a path to a FHIR resource, relative to the baseURL in the
+configuration file, and a list of query parameters. Some helpers (like `post`)
+also take a FHIR resource as a JSON object.
+
+For example, a request to find an
+[encounter by subject](https://www.postman.com/satusehat/satusehat-public/request/ef2ilan/encounter-by-subject)
+requires a URL like this:
+
+```
+https://api-satusehat-dev.dto.kemkes.go.id/fhir-r4/v1/Encounter?subject=100000030009
+```
+
+In OpenFn, we would use the `get` function, pass the path as the first argument,
+and the query parameters as the second. Like this:
+
+```js
+get('Encounter', { subject: '100000030009' });
+```
+
+The result will be written to `state.data`.
+
+## Integration Options
+
+1. **[REST APIs](https://satusehat.kemkes.go.id/platform/docs/id/postman-workshop/)**
+   that enable external services like OpenFn to _pull_ data from Satusehat, or
+   push data from external apps to Satusehat. This option is suited for
+   \_scheduled or workflows that must update data in Satusehat with external
```

Comment:
We're almost ready to go @hunterachieng  but there's something funny in this sentence. I'm not sure what it's supposed to say - would you mind fixing it?